sys/linux/auto-manual.txt

```diff
@@ -1,4 +1,11 @@
 # Copyright 2024 syzkaller project authors. All rights reserved.
 # Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
-socket$auto_nl(domain const[AF_NETLINK], type const[SOCK_RAW], proto const[NETLINK_GENERIC]) sock_nl_generic (automatic)
+socket$auto_manual_nl(domain const[AF_NETLINK], type const[SOCK_RAW], proto const[NETLINK_GENERIC]) sock_nl_generic (automatic)
+ioctl$auto_manual_sock_SIOCGIFINDEX(fd sock, cmd const[SIOCGIFINDEX], arg ptr[out, ifreq_t[ifindex]]) (automatic)
+socket$auto_manual(domain flags[socket_domain], type flags[socket_type], proto int32) sock (automatic)
+open$auto_manual(file ptr[in, filename], flags flags[open_flags], mode flags[open_mode]) fd (automatic)
+setns$auto_manual(fd fd_namespace, type flags[ns_type]) (automatic)
+getpid$auto_manual() pid (automatic)
+syz_open_procfs$auto_manual_namespace(pid pid, file ptr[in, string[procfs_namespace_file]]) fd_namespace (automatic)
+ioctl$auto_manual_NS_GET_PARENT(fd fd_namespace, cmd const[NS_GET_PARENT], arg const[0]) fd_namespace (automatic)
```

Comment:
I know I suggested to add this file.
But now I see we effectively copy lots of our manual descriptions here.
I am thinking maybe it's better to mark some of existing manual syscalls with some new attribute ("automatic-helper" or something) and enable them as well in automatic descriptions mode. This way we won't need to copy-paste more and more descriptions to this file.
@a-nogikh what do you think?

Reply:
Sounds good to me